lib/sisimai/data.rb

```diff
@@ -48,10 +48,10 @@ class Data
       addresser: RFC822Head[:addresser],
       recipient: RFC822Head[:recipient],
     }.freeze
-    ActionList = %r/\A(?:failed|delayed|delivered|relayed|expanded)\z/
+    ActionList = ['failed', 'delayed', 'delivered', 'relayed', 'expanded']
```

Comment:
__[RuboCop]__

Style/MutableConstant: Freeze mutable objects assigned to constants.


<!-- Comment by [SideCI](https://sideci.com) -->